Update readme and prepare dev env (#152)
owncloud/docs-webui: README.md

# ownCloud Web User Interface

**Table of Contents**

> * [Building the Web Userinterface Docs](#building-the-web-userinterface-docs)

Reviewer: ```suggestion
* [Building the Web User Interface Docs](#building-the-web-userinterface-docs)
```

> * [Building the Web User Interface Docs](#building-the-web-userinterface-docs)

Author: ```suggestion
* [Building the Web User Interface Docs](#building-the-web-user-interface-docs)
```

* [General Notes](#general-notes)
* [Generating the Documentation](#generating-the-documentation)
* [Target Branch and Backporting](#target-branch-and-backporting)
* [Branching Workflow](#branching-workflow)

> * [Create a New Version Branch for Web Userinterfaces](#create-a-new-version-branch-for-web-userinterfaces)

Reviewer: ```suggestion
* [Create a New Version Branch for Web User Interfaces](#create-a-new-version-branch-for-web-userinterfaces)
```

> * [Create a New Version Branch for Web User Interfaces](#create-a-new-version-branch-for-web-userinterfaces)

Author: ```suggestion
* [Create a New Version Branch for Web User Interfaces](#create-a-new-version-branch-for-web-user-interfaces)
```

> ## Building the Web Userinterface Docs

Reviewer: ```suggestion
## Building the Web User Interface Docs
```

> Please refer to the [Branching Workflow for ownCloud Server](https://github.com/owncloud/docs-webui/blob/master/docs/the-branching-workflow.md) or more information.

Reviewer: ```suggestion
Please refer to the [Branching Workflow for ownCloud Server](https://github.com/owncloud/docs-webui/blob/master/docs/the-branching-workflow.md) for more information.
```

> ## Create a New Version Branch for Web Userinterfaces

Reviewer: ```suggestion
## Create a New Version Branch for Web User Interfaces
```

> Please refer to [Create a New Version Branch for Web Userinterfaces](https://github.com/owncloud/docs-webui/blob/master/docs/new-version-branch.md) for more information.

Reviewer: ```suggestion
Please refer to [Create a New Version Branch for Web User Interfaces](https://github.com/owncloud/docs-webui/blob/master/docs/new-version-branch.md) for more information.
```
